app/controllers/orga/conferences_controller.rb

```diff
@@ -1,30 +1,14 @@
 class Orga::ConferencesController < Orga::BaseController
-  before_action :find_conference, only: [:show, :edit, :update, :destroy]
+  before_action :find_conference, only: [:show, :destroy]
+
+  def import
+    Conference::Importer.call_with(params[:file])
```

Comment:
LOL Ruby is all about calling methods, so _of course_ it calls :wink:
I am very tempted to go with `Importer.do_your_magic_with(file)` but I went with `call_with(file)`.  Because `call(file)` looks weird to me.
Okay? Or is this undermining the architecture argument?